charts/pomerium/README.md

```diff
@@ -91,11 +91,32 @@ You may force recreation of your TLS certificates by setting `config.forceGenera
 
 If you wish to provide your own TLS certificates in secrets, you should:
 
-1. turn `generateTLS` to `false`
+1. turn `config.generateTLS` to `false`
 2. specify `authenticate.existingTLSSecret`, `authorize.existingTLSSecret`, and `proxy.existingTLSSecret`, pointing at the appropriate TLS certificate for each service.
 
 All services can share the secret if appropriate.
 
+## Signing Key
+
+### Auto Generation
+
+In default configuration, this chart will automatically generate a signing key in a helm `pre-install` hook for the Pomerium proxy to sign jwt sent in responses.
+
+Upon delete, you will need to manually delete the generated secret. Example:
+
+```console
+kubectl delete secret -l app.kubernetes.io/name=pomerium
+```
+
+You may force recreation of your signing key by setting `config.forceGenerateSigningKey` to `true`. Delete already existing signing key secret first to prevent errors, and make sure you set back to `false` for your next helm upgrade command or your deployment will fail due to existing Secret.
+
+### Self Provisioned
+
+If you wish to provide your own signing key in secret, you should:
+
+1. turn `config.generateSigningKey` to `false`
```

Comment:
@tgomas this option does not seem to be documented below

Reply:
also `forceGenerateSigningKey`

Reply:
done